### PR TITLE
use JOML for vector-based normalization

### PR DESCRIPTION
**Pull request by dankfmemes**

### CODE REVIEW

**Review by CiroZDP — APPROVED** [review]
Good!

**Review by CiroZDP — APPROVED** [review]

File: src/main/java/net/opencraft/client/input/MovementInputFromOptions.java [hunk 1]
```diff
@@ -65,7 +65,6 @@ public void updatePlayerMoveState(final EntityPlayer gi) {
 
         if (movement.lengthSquared() > 0) {
             movement.normalize();
-            System.out.println("normalizing movement");
```
Comment (dankfmemes) [inline src/main/java/net/opencraft/client/input/MovementInputFromOptions.java]: Probably should remove that first 💀

Comment (CiroZDP) [reply src/main/java/net/opencraft/client/input/MovementInputFromOptions.java]: okay

**Comment by CiroZDP** [comment]
then don't merge this

File: src/main/java/net/opencraft/client/input/MovementInputFromOptions.java (hunk 1 above)
Comment (dankfmemes) [reply src/main/java/net/opencraft/client/input/MovementInputFromOptions.java]: Removed at https://github.com/OpenCraft-Studios/OpenCraft/pull/29/commits/25865a142fa2fc71fec90b3fdb284839d300dab8

**Pull request closed without merging**